src/Chapter1.hs

```diff
@@ -520,7 +521,9 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if abs(x) < abs(y)
```

Comment:
No need to add brackets to one variable 🙂 

Also, in Haskell, all functions are separated from their arguments by space 👾 

```suggestion
closestToZero x y = if abs x < abs y
```